bloges56/DateSim
# Bl interact scene switch

## Description
- Created a script that when added to an object with a collider, that object can now be interacted with to switch to a specified scene.

## How to Test
1. Go to SceneManager Scene and hit play
2. Hit start game
3. Approach the cylinder and press E to switch to the HomeScreen
4. Stop the game and go to GrayBoxMain Scene
5. In the SceneInteract component on the SceneSwitch object, set the sceneName variable to any scene you want, except the SceneManager.  Just make sure that the scene you want to load is added to the build settings.
6. Repeat steps 1-3 and now interacting with the object should load the scene you specified

## Checklist
- [x] Does getting close to the SceneSwitch object display prompt?  Does moving away remove it?
- [x] Does pressing E near the SceneSwitch object load the scene currently specified in the inspector?
- [x] Does interacting with the other object work as before?
